php-checks/src/test/resources/checks/LocalVariableShadowsClassFieldCheck.php

```diff
@@ -26,10 +33,15 @@ public function f2($param) {          // OK
     $field = 1;                         // NOK {{Rename "$field" which has the same name as the field declared at line 9.}}
   }
 
-  public function f3($field) {}         // NOK
+  public function f3($field) {          // OK
+    if (empty($field)) {
```

Comment:
@vilchik-elena Because I had a FP at some point on such code: as the original code did not use the symbol table at all, simply ignoring parameter definitions led to the conclusion that `$field` was a local variable in this case.